mslipenchuk267/parliament
# Bluetooth mock description

# What was accomplished?
- Refactored SettingsScreen.js based on tester's comments and their user experience
- Changed 'bluetooth mock' header into 'Add device'
- Separated 'Device Storage' into two different components, 'Scanned Devices' and 'Notification History'. 
- Moved 'Account' component to bottom section of screen

# How was this tested?
- Manual testing to confirm that:
    - after changing headers and text, I refreshed immediately to make sure button still has same functionality by confirming logs to make sure no bugs were created
    
# New dependencies?
- None.
